docs/data-ai/data/visualize.md

```diff
@@ -35,42 +35,152 @@ See [configure a machine](/operate/get-started/supported-hardware/) for more inf
 
```

Comment:
out-of-diff: noticed the prereq says at least one of the following and then has a single bullet. consider breaking sensor and movement sensor into separate bullets, and linking each to its respective component page instead of the configure page.